sql/src/test/java/io/crate/integrationtests/FulltextAnalyzerResolverTest.java

```diff
@@ -329,7 +329,7 @@ public void testBuiltInTokenFilters() throws Exception {
                "pattern_replace, persian_normalization, porter_stem, reverse, " +
                "russian_stem, scandinavian_folding, scandinavian_normalization, serbian_normalization, " +
                "shingle, snowball, sorani_normalization, standard, stemmer, stemmer_override, " +
-               "stop, synonym, trim, truncate, unique, uppercase, word_delimiter"));
+               "stop, synonym, trim, truncate, unique, uppercase, word_delimiter, word_delimiter_graph"));
```

Comment:
I was wondering about that as well. The vast majority of filters are not documented, so I think it doesn't make sense to document this special filter.